### PR TITLE
polish oncall instructions

### PR DESCRIPTION
**Pull request by xiaolanz**

### CODE REVIEW

File: ON-CALL.md [hunk 1]
```diff
@@ -27,14 +29,12 @@ and is performed during regular working hours for your time zone.
...
 * Help with creating the release when needed.
-* Check the schedule sheet to make sure the next on call is defined.
+* Use this on-call time to learn something new about Istio. Work with the subject matter experts and contribute to the troubleshooting guides on [istio.io](https://istio.io/help/troubleshooting.html).
 * On Friday, notify the next on-call.
-* On Tuesday, at the end of the on-call, handoff to next on-call and send a summary to istio-oncall and istio-dev. Include the stats below, that can be obtained by querying [GitHub with dates ranges:](https://help.github.com/articles/searching-issues-and-pull-requests/)
-
-* At the begining of your on-call make sure https://goo.gl/9wjRdg is updated with current snapshot numbers (of issues, PRs, etc...)
-* At the end of your on-call, update https://goo.gl/9wjRdg - the deltas will be calculated for you (bold columns) please send an email summary
+* On Tuesday, at the end of your on-call
+  * Update "blue" columns in https://goo.gl/9wjRdg - the deltas will be calculated for you (bold columns) please send an email summary
```
Comment (cmluciano) [inline ON-CALL.md]: s/please/and

Comment (xiaolanz) [reply ON-CALL.md]: PTAL

**Pull request merged**